rcsb/utils/dictionary/DictMethodEntityHelper.py

```diff
@@ -354,22 +358,21 @@ def buildContainerEntityIds(self, dataContainer, catName, **kwargs):
                                 else:
                                     refSeqIdD["dbIsoform"].append("?")
                         # else fallback to ma_target_ref_db_details
-                        else:
-                            if (dataContainer.exists("ma_target_ref_db_details")):
-                                mObj = dataContainer.getObj("ma_target_ref_db_details")
-                                for jj in range(mObj.getRowCount()):
-                                    tEntityId = mObj.getValue("target_entity_id", jj)
-                                    tDbName = mObj.getValue("db_name", jj)
-                                    tDbAccession = mObj.getValue("db_accession", jj)
-                                    tDbIsoform = mObj.getValue("seq_db_isoform", jj)
-                                    if tEntityId == entityId and tDbName in ["UNP"] and tDbAccession not in [".", "?"]:
-                                        refSeqIdD["dbName"].append("UniProt")
-                                        refSeqIdD["provSource"].append(provSourceDefault)
-                                        refSeqIdD["dbAccession"].append(tDbAccession)
-                                        if tDbIsoform:
-                                            refSeqIdD["dbIsoform"].append(tDbIsoform)
-                                        else:
-                                            refSeqIdD["dbIsoform"].append("?")
+                        elif (dataContainer.exists("ma_target_ref_db_details")):
+                            mObj = dataContainer.getObj("ma_target_ref_db_details")
```

Comment:
These changes (lines 361-375) were already committed to staging. I don't understand why this is showing up as new changes here.

Reply:
I changed the indentation by one, because I consolidated the previous
```python
else:
     if ...:
```
with:
```python
elif ...:
```

Reply:
So it's appearing as a block-wide change

Reply:
With regard to the changes you already committed earlier today—I don't know why they are showing up as new changes. It's just git being weird after having rebased, etc.

Reply:
Ok thanks.

Reply:
It looks good to me. If the tests worked, no need to revert the merge.

Reply:
OK great, thanks for checking!